core/_variables.scss

```diff
@@ -1,3 +1,14 @@
+// Breakpoints
+$medium-screen: 600px;
+$large-screen: 900px;
+
+
+// Grid Settings
+$column: 90px;
+$gutter: 30px;
+$grid-columns: 12;
+$max-width: 1200px;
```

Comment:
Agree, I actually wanted to do that but i figured someone would complain.

:fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire: